tools/dbtool.py

```diff
@@ -979,14 +979,23 @@ def configure_and_launch_multi_process_by_zonetype():
     # fmt: on
 
 
+def update_submodules():
+    # fmt: off
+    result = subprocess.run(
+        ["git", "submodule", "update", "--init", "--recursive", "--progress"], capture_output=True, text=True).stdout
+    print(result)
+    # fmt: on
+
+
 def tasks_menu():
     present_menu(
         "Maintenance Tasks",
         {
-            "1": ["Set zone IP addresses", set_external_ip_dialog],
-            "2": ["Server-wide announcement", announce_menu],
-            "3": ["Show table sizes (min 2MB)", print_db_tables_by_size],
-            # "4": [
+            "1": ["Update git submodules", update_submodules],
```

Comment:
Love having this baked into dbtool as well.  For future, we should probably update the wiki to instruct people to use it on first install and updates (even if we're not updating the submodules often)